A/AMCL/build_tarballs.jl

```diff
@@ -0,0 +1,107 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder
+
+name = "AMCL"
+version = v"2.0.1"
+
+# Collection of sources required to complete build
+sources = [
+    ArchiveSource("https://github.com/apache/incubator-milagro-crypto-c/archive/$version.tar.gz", "48fc7e0d7ad5edbfe8a779dc56f28813c25798ddfd60f1d94ccf1203eacd5645"),
+]
+
+# Bash recipe for building across all platforms
+script = """
+cd "\$WORKSPACE/srcdir/incubator-milagro-crypto-c-$version"
+""" + raw"""
+set -e
```

Comment:
We use this trick to avoid having to reference the version:
```suggestion
script = raw"""
cd $WORKSPACE/srcdir/incubator-milagro-crypto-c-*
```
:wink: 

Also, `set -e` is automatically set in the build environment

Reply:
👍 - implemented both comments.